content/en/community/hackathons/sessions/baby-steps/content/Setup_Helloworld.md

```diff
@@ -0,0 +1,345 @@
+It is **heavily** recommended that for building and developing applications with Unikraft, you use a conventional folder structure.
+Namely, for the examples that we will be going through in this session, our working directory will look like this:
+
+```console
+workdir                     ---> working directory containing applications
+`-- helloworld              ---> application directory
+    |
+    |
+    |-- main.c              ---> application code
+    |-- Makefile            ---> makefile for specifying external library dependencies
+    |-- Config.uk           ---> unikraft configuration options for the application
+    |-- Makefile.uk         ---> the application's compiler flags and source files to build
+    |
+    |
+    `-- .unikraft
+        |-- libs        ---> external libraries used by the application
+        `-- unikraft    ---> unikraft core kernel, containing all the internal libraries
+`-- nginx
+    |
+    |
+    |
+`-- python3
+    |
+    |
+    |
+...
+```
+
+Let's begin with the simplest app there is, `helloworld`.
+We first create the `workdir` folder where we will place all of our applications and clone the [`app-helloworld`](https://github.com/unikraft/app-helloworld/) repository, which contains the source files and the Makefiles necessary for bulding the `helloworld` application.
+
+```bash
+$ mkdir workdir
+
+$ cd workdir/
+
+$ git clone https://github.com/unikraft/app-helloworld helloworld
+
+$ cd helloworld/
+[...]
+
+$ ls -F
+[...]  Makefile  Makefile.uk  README.md  main.c  run-qemu-x86_64.sh* [...]
```

Comment:
```suggestion
[...]  Makefile  Makefile.uk  README.md  main.c  [...]
```
We removed the scripts and just added the commands inside the `README` files for every app, so this is not there anymore.